README.md

```diff
@@ -1,6 +1,7 @@
 # Terratest
 
 [![Maintained by Gruntwork.io](https://img.shields.io/badge/maintained%20by-gruntwork.io-%235849a6.svg)](https://gruntwork.io/?ref=repo_terratest)
+[![CircleCI](https://circleci.com/gh/gruntwork-io/terratest.svg?style=svg)](https://circleci.com/gh/gruntwork-io/terratest)
```

Comment:
```suggestion
[![CircleCI](https://circleci.com/gh/gruntwork-io/terratest.svg?style=svg&circle-token=e48019e09fc3b8bf6e0315a84048501c87c4157c)](https://circleci.com/gh/gruntwork-io/terratest)
```

Reply:
@Etiene thank you. Even with this above, https://circleci.com/gh/gruntwork-io/terratest still doesn't have anonymous access ...